src/kata5.reachDestination.js

```diff
@@ -1,3 +1,5 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+	return `I should be there in ${Math.ceil((distance / speed) * 2) / 2} hours`;
```

Comment:
The returned string is grammatically incorrect if the arrival time is 1. i.e. it should be `"I should be there in 1 hour"` but currently it returns `"I should be there in 1 hours"`.

To fix this, and also improve readability, we can take out the arrival time calculation and assign it to a variable. i.e.
```
const arrival = Math.ceil((distance / speed) * 2) / 2
return `I should be there in ${arrival} hours`;
```
then, we can use this arrival time to conditionally add the `'s'` at the end of `'hour'`, e.g.
```
const arrival = Math.ceil((distance / speed) * 2) / 2
return `I should be there in ${arrival} hour${arrival === 1 ? "" : "s"}`;
```